mcp/include/command_struct.h

```diff
@@ -68,26 +71,37 @@
 
 /* Need to undef I here in for source files that utilize complex.h */
 #undef I
+/**
+ * @brief PID control gains structure
+ * 
+ * 
+ */
 struct GainStruct {
-  float P;
-  float I;
-  float D;
-  float SP;
-  float DB; // Deadband the integral term
-  float PT; // position velocity gain
-  float F; // Current offset to overcome static friction.
+  float P; /** proportional term */
```

Comment:
I believe Doxygen markup for documenting struct/union members is `/**< ... */` (note the `<` after `/**`). It looks like these can be easily fixed with
```
sed 's/\/\*\* \(.*\) \*\/$/\/**< \1 *\//' command_struct.h
```